app/models/user.rb

```diff
@@ -77,6 +79,20 @@ def secure_password
     end
   end
 
+  def self.from_omniauth(access_token)
+    data = access_token.info
+    user = User.where(:email => data["email"]).first
+
+    # Uncomment the section below if you want users to be created if they don't exist
```

Comment:
Confirm that we don't want users to be created if they don't exist 💯 . Got this exactly right